Autofac.Diagnostics.DotGraph.sln

```diff
@@ -0,0 +1,56 @@
+﻿

```

Comment:
Hmmm. This might take some doing. `dotnet sln` doesn't support adding solution items. I'll figure it out.

Reply:
Out of curiosity... are these _valuable_ in the other projects? Like, how often do you work with the `.editorconfig` or whatever in the other projects? Are we doing it "just because" or is it helpful?

Reply:
I tend to edit the Directory.Build.props and editorconfig directly from VS. If it's a pain from the Mac don't worry about it, I can do it myself once this is merged.

Reply:
OK, all you poor VS saps have your solution items. 😆 And now my VS instance is up to date again. Man, I remember using VS. It's been so long. 😢